src/hip_ra_x/hip_ra_x.py

```diff
@@ -41,6 +41,8 @@
 from geophires_x.Units import TimeUnit
```

Comment:
Tests are failing because pre-commit was not run - https://github.com/NREL/GEOPHIRES-X/pull/115/checks; - per test output, run `pre-commit run --all-files`. (Set up pre-commit first if you haven't already, per step 3 of Local Setup instructions: https://github.com/NREL/GEOPHIRES-X/blob/main/CONTRIBUTING.rst#local-setup)

Reply:
To elaborate - the reason pre-commit is failing is the imports are not sorted, which is a standard formatting rule. Tests enforce formatting to keep diffs clean from formatting changes so PRs are more reviewable and focused on content rather than style

Reply:
The "pre-commit" was new to me - I must have missed instructions about installing it. From what I understand, now that I have it installed, it will run when I run git commit from the command line... so I won't have to run it manually as I did for this.... it will also run if the commit is done within the PyCharm GUI.